jenkins/ci.opensuse.org/diskimage-builder.yaml

```diff
@@ -0,0 +1,86 @@
+- job:
+    name: 'diskimage-builder'
+    project-type: freestyle
+
+    triggers:
+        - timed: '@daily'
+
+    retry-count: 3
+
+    concurent: false
+
+    build-discarder:
+    num-to-keep: 10
+    days-to-keep: -1
+
+    parameters:
+        - bool:
+            name: BOOT_DIB_IMAGE
+            default: false
+            description: Boot the generated image using QEMU/KVM.
+
+    node: cloud-trackupstream
+
+    builders:
+        - shell: |
+            #!/bin/bash
+            set -e
+
+            cleanup() {
+                # Clean up in case the last job was broken
+                rm -rf /tmp/diskimage-builder /tmp/dib-utils
+                rm -f *.qcow2 diskimage-builder-boot.log
+            }
+
+            trap cleanup EXIT
+
+            # Start fresh
+            cleanup
+            git clone https://github.com/openstack/diskimage-builder.git /tmp/diskimage-builder
+            git clone https://github.com/openstack/dib-utils.git /tmp/dib-utils
+
+            # Prefer the latest tools in PATH
+            export PATH=/tmp/diskimage-builder/bin:/tmp/dib-utils/bin:/usr/bin:/bin:${PATH}
+
+            # We are going to boot our image but only if we can do it on the
+            # running host
+            ${BOOT_DIB_IMAGE} && zypper in -y qemu-kvm &> /dev/null || true
+
+            # DIB elements to use
+            DIB_ELEMENTS="opensuse vm"
+
+            # Make sure all releases build
+            # 13.2 rootfs images have vanished from the mirror so take it out
+            # of the rotation for now
+            OPENSUSE_RELEASE_BUILDS=(13.1 42.1)
```

Comment:
using a matrix job for that would be better. That way we see which releases are broken. See the openstack-cleanvm job